### PR TITLE
Includes a function to help the ground support downloading sat images.

### PR DESCRIPTION
**Pull request by JuliaWindmiller1987**
The function downloads a GOES image with predefinded corner coordinates and saves it under a comprehensive file name.

### CODE REVIEW

File: orcestra/sat.py [hunk 1]
```diff
@@ -215,3 +216,42 @@ def get_track_for_day(self, day):
...
+    else:
+        raise ValueError(
+            "Invalid option for layer. Use 'vis' for visible or 'inf' for infrared."
+        )
+
+    bbox = "5,-60,20,-40"
+
+    url = f"https://wvs.earthdata.nasa.gov/api/v1/snapshot?REQUEST=GetSnapshot&TIME={time}&BBOX={bbox}&CRS=EPSG:4326&LAYERS={layer},Coastlines_15m&WRAP=x,x&FORMAT=image/jpeg&WIDTH=876&HEIGHT=652&ts=1725710721315"
+
+    filename = f"{time}_{bbox}_{layer}.jpg"
```
Comment (lkluft) [inline orcestra/sat.py:243]: Commas in filenames are a bit annoying, you may want to replace them in the filename
```suggestion
    filename = f"{time}_{bbox}_{layer}.jpg".replace(",", "")
```

File: orcestra/sat.py [hunk 2]
```diff
@@ -215,3 +216,42 @@ def get_track_for_day(self, day):
...
+
+    if folder_path:
+        if not os.path.exists(folder_path):
+            os.makedirs(folder_path)
```
Comment (lkluft) [inline orcestra/sat.py:247]:
```suggestion
        os.makedirs(folder_path, exist_ok=True)
```

**Review by lkluft — CHANGES_REQUESTED** [review]
Thanks for adding this! :) I made two minor comments that you may want to address before merging

File: orcestra/sat.py (hunk 2 above)
Comment (JuliaWindmiller1987) [reply orcestra/sat.py:247]: Ah, nice, I did not know this!

**Pull request merged**